src/cmake/axom-config.cmake.in

```diff
@@ -162,7 +162,8 @@ if(NOT AXOM_FOUND)
   foreach(_component ${AXOM_COMPONENTS_ENABLED})
     set_property(TARGET axom
                  PROPERTY INTERFACE_LINK_LIBRARIES
-                 axom::${_component})
+                 axom::${_component}
+                 APPEND)
```

Comment:
Note: without `APPEND`, the `axom` target was only depending on the last component, e.g. `axom::multimat`

Reply:
Oh good catch.